src/lib/css-tag.ts

```diff
@@ -12,7 +12,8 @@ found at http://polymer.github.io/PATENTS.txt
 /**
  * Whether the current browser supports `adoptedStyleSheets`.
  */
-export const supportsAdoptingStyleSheets =
+export const supportsAdoptingStyleSheets = (window.ShadowRoot) &&
```

Comment:
Think we need to change this to `('adoptedStyleSheets' in window.ShadowRoot.prototype)`, instead of `('adoptedStyleSheets' in Document.prototype)`. This makes sure it'll be false when ShadyDOM is enabled.

Reply:
Seems like this change should be needed to get the test running under the polyfill to run?

Reply:
@justinfagnani had an issue with this. To quote him:

> We need to keep this checks against Document, since we have developers that delete the properties on Document.prototype to disable the use of adopted style sheets so they can edit in DevTools.